### PR TITLE
platform-configs: Add support for VP3XX platforms

### PR DESCRIPTION
**Pull request by JanPrusinowski**

### CODE REVIEW

File: platform-configs/include/protectli-vp32xx.robot [hunk 1]
```diff
@@ -0,0 +1,15 @@
+*** Settings ***
+Resource    protectli-common.robot
+
+
+*** Variables ***
+${FLASH_SIZE}=                  ${16*1024*1024}
+${VERIFIED_BOOT_SUPPORT}=       ${TRUE}
+${IPXE_BOOT_ENTRY}=             Network Boot and Utilities
```
Comment (macpijan) [inline platform-configs/include/protectli-vp32xx.robot:8]: @JanPrusinowski @SebastianCzapla It's already defined in protectli-common, right?

File: platform-configs/include/protectli-vp32xx.robot [hunk 2]
```diff
@@ -0,0 +1,15 @@
+*** Settings ***
+Resource    protectli-common.robot
+
+
+*** Variables ***
+${FLASH_SIZE}=                  ${16*1024*1024}
+${VERIFIED_BOOT_SUPPORT}=       ${TRUE}
```
[the remark this answers is not in the document]
Comment (macpijan) [reply platform-configs/include/protectli-vp32xx.robot:7]: Unless you have identified some tests/features from protectli-pro file, not applicable for VP3000 series.

[the remark this answers is not in the document]
Comment (JanPrusinowski) [reply platform-configs/include/protectli-vp32xx.robot:7]: That's what I thought too so I actually added it in the branch I am working atm on this platform: https://github.com/Dasharo/open-source-firmware-validation/pull/546/commits/822966286ff04e26fcaaac4fe93be6b9ab540457
But I can add it here.

File: platform-configs/include/protectli-vp32xx.robot (hunk 1 above)
Comment (JanPrusinowski) [reply platform-configs/include/protectli-vp32xx.robot]: You are correct. I have removed those entries in: e513d6b99380920e382407728ea9a30bb19a2a9f

File: platform-configs/include/protectli-vp32xx.robot (hunk 2 above)
[the remark this answers is not in the document]
Comment (JanPrusinowski) [reply platform-configs/include/protectli-vp32xx.robot]: Added protectli-pro resource in: e513d6b99380920e382407728ea9a30bb19a2a9f

File: platform-configs/include/protectli-vp32xx.robot [hunk 3]
```diff
@@ -0,0 +1,14 @@
+*** Settings ***
+Resource    protectli-common.robot
```
Comment (macpijan) [inline platform-configs/include/protectli-vp32xx.robot:2]: @JanPrusinowski FYI, I have already tested it before, and the order of these includes matters if we want to allow overriding variables. The order should be reversed actually, as in the `protectli-vp66xx.robot`  for instance:

```
*** Settings ***
Resource    protectli-pro.robot
Resource    protectli-common.robot
```

Comment (macpijan) [reply platform-configs/include/protectli-vp32xx.robot:2]: If I am right, in current order, the variable defined in `protectli-common`  would be used, not the one from `protectli-pro`. Can you please verify that as well? Maybe also add explanation to https://github.com/Dasharo/open-source-firmware-validation/blob/develop/docs/adding-new-platforms.md if this ordering really matters?

Comment (JanPrusinowski) [reply platform-configs/include/protectli-vp32xx.robot]: You are right. I have also checked it. When Robot Frameworks looks for for example a key then it searches from top to bottom looking into resources in that order until it finds a key - then it skips searching the rest.
Reversed the order: https://github.com/Dasharo/open-source-firmware-validation/pull/544/commits/fb59cb4b22c73dbb4bc27d50a0a1503776e286f5

**Pull request merged**